src/controllers/FeedController.js

```diff
@@ -89,9 +92,23 @@ module.exports = {
 
       const result = await getFeedByIdService.execute({ id });
 
-      res.status(200).json(result);
+      return res.status(200).json(result);
+    } catch (err) {
+      return next(err);
+    }
+  },
+
+  async approveFeeds(req, res, next) {
+    try {
+      const { id } = req.params;
+
+      const { is_pendent } = req.body;
+
+      const result = await approveFeedService.execute({ id, is_pendent });
+
+      return res.status(201).json(result);
```

Comment:
@gumeeee status code 201 é usado para quando for criar um registro, nesse caso creio que o status code correto seria 200